rustler_tests/native/rustler_test/src/test_env.rs

```diff
@@ -65,5 +64,5 @@ pub fn sublists<'a>(env: Env<'a>, args: &[Term<'a>]) -> NifResult<Term<'a>> {
         });
     });
 
-    Ok(atom::ok().to_term(env))
+    Ok(atom::ok())
```

Comment:
Maybe just return `Atom` here?

Reply:
This function uses `?` so it has to return a `Result`.